packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -205,6 +205,7 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
     apiKey,
     developerPlatformClient,
   }: ExtensionDeployConfigOptions): Promise<{[key: string]: unknown} | undefined> {
+    // To ensure this is not a breaking change we will delete this upload line on a second pass after we've ensured the new file upload process from core is working.
```

Comment:
>so we'll do both uploads GCS+this one?
Only till I am able to verify prod bucket access works so that would be till core is deployed and this cli version is deployed.
>Ping me on the cleanup PR as well, there is a few things we can remove after deleting this line :)
Absolutely will do, can also add you as a reviewer on my cleanup PR! :)